src/app/shared/components/template/components/navigation-bar/navigation-bar.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="navigation-bar-wrapper">
+  <div
```

Comment:
Generally it's best to use divs only when no other native components fit - as this is a link it probably is better to use an `a` tag (there's quite a few benefits in terms of accessibility and code quality using more fundamental elements.

Additionally angular provides a lot of tools for managing navigation so really we don't want to be using manual click methods when we can use [routerLink](https://angular.io/api/router/RouterLink), and we don't need to manually track if a route is already active because angular provides [routerLinkActive](https://angular.io/api/router/RouterLinkActive)